src/components/app/App.jsx

```diff
@@ -31,6 +32,7 @@ export default function App() {
         <PageRoute exact path="/:enterpriseSlug/search" component={SearchPage} />
         <PageRoute exact path="/:enterpriseSlug/course/:courseKey" component={CoursePage} />
         <PageRoute exact path="/:enterpriseSlug/licenses/:activationKey/activate" component={LicenseActivationPage} />
+        <PageRoute exact path="/:enterpriseSlug/skills-quiz" component={SkillsQuizPage} />
```

Comment:
Shouldn't the pathname be like `skillsQuiz`

Reply:
I think it is preferred to use hyphenated in words in URLS instead of using camelcase.